### PR TITLE
Added automatic label type calculation

### PR DESCRIPTION
**Pull request by nhoffmann-newrelic**
This is a breaking change, every code that was using the parameter `label_type` will now have to remove it

### CODE REVIEW

**Review by YehezkelShay — APPROVED** [review]

**Comment by ykriger-newrelic** [comment]
@nhoffmann-newrelic, we have to update the README.md, notebook examples and guided activity accordingly.

**Comment by ykriger-newrelic** [comment]
For the breaking changes, maybe it makes sense to keep the label_type as input but ignore it and put some warning.
@nhoffmann-newrelic

**Comment by YehezkelShay** [comment]
@nhoffmann-newrelic, I think that instead of talking in terms of `numeric` and `categorical` we need to talk in `classification `and `regression`. 
This will the right way and also it will make the work easier:

- classification- pie plot
- regression- bar plot

File: src/ml_performance_monitoring/monitor.py [hunk 1]
```diff
@@ -65,7 +65,6 @@ def __init__(
         send_data_metrics=False,
         features_columns: Optional[List[str]] = None,
         labels_columns: Optional[List[str]] = None,
-        label_type: Optional[str] = None,
```
Comment (YehezkelShay) [inline src/ml_performance_monitoring/monitor.py:68]: I thinks that we don't need to remove it in the next few week so we won't break the code to the users

**Comment by nhoffmann-newrelic** [comment]
> please bump the library version in [pyproject.toml](https://github.com/newrelic-experimental/ml-performance-monitoring/blob/main/pyproject.toml)

done

File: src/ml_performance_monitoring/monitor.py (hunk 1 above)
Comment (nhoffmann-newrelic) [reply src/ml_performance_monitoring/monitor.py]: done

**Review by YehezkelShay — APPROVED** [review]
please update Amir to remove the label_type from the guided installation

**Pull request merged**